### ISSUE TITLE
Turn list into list of bubbles

### ISSUE THREAD

**Issue by phdoerfler** [foambubble/foam#694]
TL;DR:

```
- this
- that
- dooblydoo
```

↓

```
- [[this]]
- [[that]]
- [[dooblydoo]]
```

I imagine selecting the entire list (or some part thereof), then issuing a "New Notes / Bubbles from Selection" command and be done.

---

I often start with a list where I quickly brainstorm and jot down ideas. Often once I'm done I want to turn every list item into its own bubble. Currently I have to select it manually, then use the `vscodeMarkdownNotes.newNoteFromSelection` command and enter a title. It's a lot better than nothing but really, I want to be able to do this for all of these list items at simultaneously.

Thoughts? Also, should I rather file this as a feature request in the Markdown Notes Github?

**Comment by riccardoferretti** [comment]
this might not be satisfactory to your use case, but here is how I would go about this:
1. select "- "
2. press CMD+D to select all occurrences 
3. select rest of line and type `[[` to turn them into bubbles

Because this feels more like an editing feature than a "connection" feature I don't believe it needs to live in Foam "core", but I am open to disagreements here if others feel differently.

But because of the "decent" workaround and my belief this doesn't need to live in Foam, I will close it for now until a good case is made for the feature

**Comment by phdoerfler** [comment]
I agree, that would be a decent workaround if it handled list items with spaces and capitalisation correctly. For me writing, e.g., `[[This is another thought]]` then Cmd-clicking on it results in a mess. It creates a markdown file in the root directory and not the directory the file I’m currently editing is in, for instance. This is in contrast to Markdown Notes which does respect that and creates working links and good titles and filenames.

**Comment by riccardoferretti** [comment]
the problem of cmd+click is due to another issue, which will be addressed by https://github.com/foambubble/foam/issues/566

**Comment by phdoerfler** [comment]
Ah wonderful!
